src/common/dpp_tcp.c

```diff
@@ -583,7 +583,7 @@ int dpp_relay_rx_gas_req(struct dpp_global *dpp, const u8 *src, const u8 *data,
 			 size_t data_len)
 {
 	struct dpp_relay_controller *ctrl;
-	struct dpp_connection *conn, *found = NULL;
+	struct dpp_connection *conn = NULL, *found = NULL;
```

Comment:
nice... this is why I also don't like doing this... my personal style is to put each declare on its own line.